app/controllers/nominations/nominate_induction_coordinator_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class Nominations::NominateInductionCoordinatorController < ApplicationController
+  def new
+    @token = params[:token]
+    @nomination_email = NominationEmail.find_by(token: @token)
+
+    if @nomination_email.nil?
+      redirect_to link_invalid_nominate_induction_coordinator_path
+    elsif @nomination_email.expired?
+      redirect_to link_expired_nominate_induction_coordinator_path(school_id: @nomination_email.school_id)
+    elsif @nomination_email.school.fully_registered?
```

Comment:
Is it worth changing this to registered? fully_registered? was good when we had partially_registered? now it just raises unnecessary questions.